examples/internal/proto/examplepb/stream.swagger.json

```diff
@@ -116,6 +116,37 @@
           "StreamService"
         ]
       }
+    },
+    "/v1/example/download": {
+      "get": {
+        "operationId": "StreamService_Download",
+        "responses": {
+          "200": {
+            "description": "A successful response.(streaming responses)",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "result": {
+                  "$ref": "#/definitions/apiHttpBody"
+                },
+                "error": {
+                  "$ref": "#/definitions/runtimeStreamError"
+                }
+              },
```

Comment:
In this case we'll need to override the swagger schema, but again, lets do that in a separate issue.